### DISCUSSION TITLE
What is your governance model?

### DISCUSSION THREAD

**Discussion by alehar9320** [taiga-family/taiga-ui#1676] [Q&A] [answered]
Hi!

Congratulations on managing to create one of the most feature-rich and complete component libraries for Angular. What is your governance model? 

E.g. who decides what should be included and what should not be? 

Is there a roadmap or grander vision?

_The reason for asking is that my team may be interested in using and contributing to Taiga UI, but wish to make sure that we do not end up having to create an isolated fork of the code._

**Comment by waterplea** [comment, marked as answer]
Hello and thank you for kind words! We are happy that our library is helping developers around the world. At the moment, I'm leading the development of Taiga UI, deciding on priorities and architecture, with help of our team of 3 other engineers. 

To understand my decision making, here's some info about this library's role in our company. It is used by several dozens internal projects as well as all external Angular based products. So most importantly, it is meant to solve their needs. We have internal bug tracker, but it's mostly empty since we push internal customers to file issues here, unless they concern a small internal portion of the library. So everything is pretty much out in the open.

Our approach to fulfilling those needs was always to make things possible through flexibility, thoughtful decomposition and generic data, rather than taking care of particular use cases. We are open to contributions and make sure they follow the same concept.

External issues might have lower priorities, but are still treated fairly — the benefit of open-source is a different set of perspectives. The things community needs can easily become something our own projects require in future so we try to anticipate it by solving those tickets. Our team is rather small so lesser priority issues might accumulate a little but we try not to leave important stuff hanging.

Overall, I believe there should be no need to manage separate fork. Maybe personal addon package or two built upon foundations from Taiga UI, like we do internally.
